BAZProjectC1/Components/CarouselMovies/movieCarouselCell.swift

```diff
@@ -0,0 +1,100 @@
+        gradient.endPoint = CGPoint(x: 0.5, y: 0.9)
+        backView.layer.insertSublayer(gradient, at: 0)
+        return backView
+    }()
+    
+    public lazy var movieTitle: UILabel = {
+        let frame = CGRect(x: 0, y: 0, width: 0, height: 0)
+        let movieTitle = UILabel(frame: frame)
+        movieTitle.translatesAutoresizingMaskIntoConstraints = false
+        movieTitle.text = "movieTitle"
+        movieTitle.numberOfLines = 0
+        movieTitle.adjustsFontSizeToFitWidth = true
+        movieTitle.sizeToFit()
+        movieTitle.textColor = UIColor.appColorWhitePrimary
+        movieTitle.textAlignment = .left
+        return movieTitle
+    }()
+    
+    public lazy var movieRanking: UILabel = {
+        let frame = CGRect(x: 0, y: 0, width: 0, height: 0)
+        let movieRanking = UILabel(frame: frame)
+        movieRanking.translatesAutoresizingMaskIntoConstraints = false
+        movieRanking.text = "movieRanking"
+        movieRanking.numberOfLines = 0
+        movieRanking.adjustsFontSizeToFitWidth = true
+        movieRanking.sizeToFit()
+        movieRanking.textColor = UIColor.appColorWhitePrimary
+        movieRanking.textAlignment = .left
+        return movieRanking
+    }()
+    
+    private lazy var posterImage: UIImageView = {
+        let posterImage = UIImageView(frame: contentView.bounds)
+        posterImage.layer.cornerRadius = RounderBorderStyleForView.rounded.rawValue
+        posterImage.rounderCorners()
+        return posterImage
+    }()
+    
+    private lazy var movieRankingImage: UIImageView = {
+        let frame = CGRect(x: 0, y: 0, width: 16, height: 16)
+        let movieRankingImage = UIImageView(frame: frame)
+        movieRankingImage.translatesAutoresizingMaskIntoConstraints = false
+        movieRankingImage.image = UIImage(named: "startRanking")?.withTintColor(UIColor.appColorYellowPrimary)
+        return movieRankingImage
+    }()
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupUI()
+    }
+    
+    func setupUI() {
+        contentView.addSubview(posterImage)
+        contentView.addSubview(backView)
+        contentView.addSubview(movieTitle)
+        contentView.addSubview(movieRankingImage)
+        contentView.addSubview(movieRanking)
+        movieRankingImage.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -8).isActive = true
+        movieRankingImage.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16).isActive = true
+        movieRankingImage.widthAnchor.constraint(equalToConstant: 24).isActive = true
+        movieRankingImage.heightAnchor.constraint(equalToConstant: 24).isActive = true
+        movieRankingImage.heightAnchor.constraint(equalTo: movieRankingImage.widthAnchor, multiplier: contentView.frame.size.height / contentView.frame.size.width).isActive = true
```

Comment:
Qué significa la división `contentView.frame.size.height / contentView.frame.size.width`. 
Considera escribir una constante local con un nombre descriptivo para este valor.
Por ejemplo:
```suggestion
        let someDescriptiveNameHere = contentView.frame.size.height / contentView.frame.size.width
        movieRankingImage.heightAnchor.constraint(equalTo: movieRankingImage.widthAnchor, multiplier: someDescriptiveNameHere).isActive = true
```